configuration/concepts/concepts.csv

```diff
@@ -1,6 +1,6 @@
 Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:km,Short name:en,Short name:km,Description:en,Description:km,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1100
-,,,WEIGHT,WEIGHT,Weight,ទម្ងន់,,,Misc,Numeric,,,,,,,,,,,
-,,,HEIGHT,HEIGHT,Height,កំពស់,,,Misc,Numeric,,,,,,,,,,,
+,,,WEIGHT,WEIGHT,Weight,ទម្ងន់ (ក្រាម),,,Misc,Numeric,,,,,gram,,,0,,,

```

Comment:
Here we are modifying the existing WEIGHT for the whole app?
Is that something we want, no kilogram anymore to record the weight of patients?
If the use case is to have a weight in gram and a weight in kilograms, we should create 2 concepts.